metadata.yaml

```diff
@@ -8,4 +8,10 @@ tags:
 requires:
   reverseproxy:
     interface: http
-series: ['xenial']
\ No newline at end of file
+resources:
+    dhparam:
+        type: file
+        filename: dhparam.pem
```

Comment:
Ok, I did not know that, awesome! That seems like a good solution, it will also speed up the deployment time.

Make sure to document the command used to generate that file, and maybe a link to SE or something that explains that reusing params is ok.